contractcourt/commit_sweep_resolver.go

```diff
@@ -24,6 +24,11 @@ import (
 // version of the commitment transaction. We can sweep this output immediately,
 // as it doesn't have a time-lock delay.
 type commitSweepResolver struct {
+	// localChanCfg is used to provide the resolver with the keys required
+	// to identify whether the commitment transaction was broadcast by the
+	// local or remote party.
+	localChanCfg channeldb.ChannelConfig
```

Comment:
🙃

Reply:
Is this OK? I thought this is more or less what you suggested. Alternatively what we can do is extract just the fields we need like basepoints etc.

Reply:
yeah just feel like the info "whether it's a local commit" has popped up in multiple places and maybe it's something you could fix in your dyn series so we don't need to decide it again.